fuse/ipns/ipns_unix.go

```diff
@@ -8,28 +8,22 @@ import (
 	"errors"
 	"io"
 	"os"
-	"path/filepath"
 	"time"
 
 	fuse "github.com/jbenet/go-ipfs/Godeps/_workspace/src/bazil.org/fuse"
 	fs "github.com/jbenet/go-ipfs/Godeps/_workspace/src/bazil.org/fuse/fs"
-	proto "github.com/jbenet/go-ipfs/Godeps/_workspace/src/code.google.com/p/goprotobuf/proto"
 	"github.com/jbenet/go-ipfs/Godeps/_workspace/src/golang.org/x/net/context"
 	eventlog "github.com/jbenet/go-ipfs/thirdparty/eventlog"
 
 	core "github.com/jbenet/go-ipfs/core"
-	chunk "github.com/jbenet/go-ipfs/importer/chunk"
-	mdag "github.com/jbenet/go-ipfs/merkledag"
+	nsfs "github.com/jbenet/go-ipfs/ipnsfs"
+	dag "github.com/jbenet/go-ipfs/merkledag"
 	ci "github.com/jbenet/go-ipfs/p2p/crypto"
-	path "github.com/jbenet/go-ipfs/path"
 	ft "github.com/jbenet/go-ipfs/unixfs"
-	uio "github.com/jbenet/go-ipfs/unixfs/io"
-	ftpb "github.com/jbenet/go-ipfs/unixfs/pb"
 	u "github.com/jbenet/go-ipfs/util"
-	lgbl "github.com/jbenet/go-ipfs/util/eventlog/loggables"
 )
 
-const IpnsReadonly = true
+var DAGSERV dag.DAGService
```

Comment:
whoops. debugging, thus the all caps (I name things ugly when they are supposed to temporary)